assets/js/modules/adsense/components/dashboard/DashboardSummaryWidget.js

```diff
@@ -51,8 +51,11 @@ function DashboardSummaryWidget() {
 	} = useSelect( ( select ) => {
 		const metrics = [ 'EARNINGS', 'PAGE_VIEWS_RPM', 'IMPRESSIONS' ];
 
+		const referenceDate = select( CORE_USER ).getReferenceDate(); // The issues is the reference date needs to be offset to match the data...
```

Comment:
A bit of an oddly worded comment here which doesn't really provide useful context, so let's remove it.
```suggestion
		const referenceDate = select( CORE_USER ).getReferenceDate();
```